libcaliptra/examples/generic/main.c

```diff
@@ -379,6 +379,24 @@ int rt_test_all_commands()
         printf("ECDSA384 Verify: OK\n");
     }
 
+    // LMS_VERIFY
+    struct caliptra_lms_verify_req lms_req = {};
+
+    status = caliptra_lms_verify(&lms_req, false);
+
+    // Not testing for full success
+    // Instead, just want to see it give the right ECC-specific error
+    // This still proves the FW recognizes the message and request data and got to the right ECC code
```

Comment:
References to ECC in the comment that look copied from above.

Reply:
Thanks for catching that! Just pushed a fix